themes/ddbasic/sass/components/ding-tabroll.scss

```diff
@@ -0,0 +1,208 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?